bin/crew

```diff
@@ -1863,7 +1863,7 @@ def sysinfo_command(args)
     - Chromebrew prefix: `#{CREW_PREFIX}`
     - Chromebrew libdir: `#{CREW_LIB_PREFIX}`
 
-    - Last update in local repository: `#{`git show -s --format='#{git_commit_message_format}' '#{CREW_LIB_PATH}'`.chomp}`
+    - Last update in local repository: #{`git show -s --format='#{git_commit_message_format}' '#{CREW_LIB_PATH}'`.chomp}
```

Comment:
Is the outer `#{...}` necessary?

Reply:
Seems this syntax highlight of github is a bit misleading😅, maybe this more clear:
```ruby
"- Last update in local repository: #{IO.popen("git show -s --format='#{git_commit_message_format}' '#{CREW_LIB_PATH}").read.chomp}"